### ISSUE TITLE
How to sunset old opportunity pages?

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#1216]
What do we do with pages that are no longer relevant?

Example: the old Fellowships landing page https://foundation.mozilla.org/opportunity/fellowships/. We no longer direct people here and content will become out of date so we don't want people to find it. 

Can we archive/delete it? Can we redirect https://foundation.mozilla.org/opportunity/fellowships/ to https://foundation.mozilla.org/fellowships/?

Similar to https://foundation.mozilla.org/opportunity/internet-health-report/ and I'm sure there will be others.

**Comment by alanmoo** [comment]
We can set up redirects, once #937 is unblocked.

**Comment by kristinashu** [comment]
Fellowships redirects will be done in https://github.com/mozilla/foundation.mozilla.org/issues/1450 but IHR still needs to be done.

**Comment by alanmoo** [comment]
Where does https://foundation.mozilla.org/opportunity/internet-health-report/ need to go to?

**Comment by kristinashu** [comment]
http://internethealthreport.org/
